app/main.py

```diff
@@ -14,4 +14,12 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    return phrase.lower().count(letter.lower())
+
+
+# Test cases
+count_occurrences("letter", "t")
+count_occurrences("abc", "a")
+count_occurrences("abc", "d")
+count_occurrences("ABC", "a")
```

Comment:
You have tests in the `tests` package, you don't need function calls here. Remove them, please

Reply:
done